apps/editing-toolkit/tsconfig.json

```diff
@@ -42,6 +42,7 @@
 		{ "path": "../../packages/launch" },
 		{ "path": "../../packages/tour-kit" },
 		{ "path": "../../packages/plans-grid" },
-		{ "path": "../../packages/i18n-utils" }
+		{ "path": "../../packages/i18n-utils" },
+		{ "path": "../../packages/viewport" }
```

Comment:
I think you mean `../../packages/viewport-react`.

But it won't work, as `@automattic/viewport-react` is a JavaScript package that doesn't export types.

They have been declared in `packages/tour-kit/src/types-patch.d.ts`, but I believe we should be able to move them to `packages/viewport-react`. I'll try that.

Reply:
Yes, it must be `@automattic/viewport-react`. We've changed that in one of the more recent commits but we forgot to update the refs.

Reply:
Thanks @scinos, it would be great if types were in `packages/viewport-react`.

Reply:
I've pushed a commit with moving the types to `packages/viewport-react` and it seems to work!

At least it now fails somewhere else:

![image](https://user-images.githubusercontent.com/975703/145015767-083bebdf-ad43-446e-a167-2c17fd2c91cc.png)

Reply:
Thanks so much, @scinos! I fixed the other TS errors and all checks are passing 👍